packages/iTwinUI-react/src/core/Table/TablePaginator.tsx

```diff
@@ -246,7 +246,8 @@ export const TablePaginator = (props: TablePaginatorProps) => {
 
   const hasNoRows = totalPagesCount === 0;
   const showPagesList = totalPagesCount > 1 || isLoading;
-  const showPageSizeList = pageSizeList && onPageSizeChange && !!totalRowsCount;
+  const showPageSizeList =
+    pageSizeList && !!onPageSizeChange && !!totalRowsCount;
```

Comment:
this formatting might need to be reverted

Reply:
How? Save without formatting?

Reply:
And it puts it same way...